lib/bugsnag/integrations/que.rb

```diff
@@ -1,3 +1,5 @@
+require 'que'
```

Comment:
It looks like the addition of this line causes spec/integrations/que_spec.rb to fail. If you intend to keep this line, you'll likely need to add que to the test group of the root Gemfile and possibly remove the manual Que mock in que_spec.rb.